__tests__/functional/tagged-format.test.ts

```diff
@@ -198,7 +198,10 @@ describe("tagged format", () => {
   `("Throws if BigInt value is $testCase", async ({ input }) => {
     expect.assertions(2);
     try {
-      const result = await client.query(fql`${input}`);
+      const result = await client.query({
+        query: "foo",
+        arguments: { foo: input },
+      });
```

Comment:
using the template format meant the encoding would happen inside of `TemplateQueryBuilder.toQuery` which broke the test since it's not rethrown as a ClientError. I didn't want to change what we were doing too much.

Reply:
i see thanks

Reply:
alternative is to do a pure check that
```javascript
expects(client.query(fql`input`)).throws()
```